FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureBlobJobStore.cs

```diff
@@ -66,7 +67,72 @@ public class AzureBlobJobStore : IJobStore
             _renewLockTimer.Elapsed += async (sender, e) => await RenewJobLockLeaseAsync();
         }
 
-        public async Task<bool> AcquireJobLock(CancellationToken cancellationToken = default)
+        public async Task<Job> AcquireJobAsync(CancellationToken cancellationToken = default)
+        {
+            var lockAcquired = await TryAcquireJobLockAsync(cancellationToken);
+            if (!lockAcquired)
+            {
+                _logger.LogError("Start job conflicted. Failed to acquire job lock.");
```

Comment:
Nit: Can we update the log content here and the one in the unit test to keep consistent (`Another job is ...`)?

Reply:
Fixed. Thanks